src/thor/unidirectional_astar.cc

```diff
@@ -210,8 +210,8 @@ inline bool UnidirectionalAStar<expansion_direction, FORWARD>::ExpandInner(
   // Skip shortcut edges for time dependent routes, if no access is allowed to this edge
```

Comment:
all of the stuff we did in here to get `path_distance` set properly is good stuff but ends up being unused in this pr. and it should be noted that when bidirectional a* degenerates to unidirectional in practice we probably need to handle these same problems with the destination edge and path_distance.. probably.. but didnt check